src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApprovedConsentOrderAboutToSubmitHandlerTest.java

```diff
@@ -141,7 +141,7 @@ void given_case_when_NotPaperApplication_then_shouldNotTriggerConsentOrderApprov
     }
 
     @Test
-    void givenCase_whenNoPendsion_thenShouldUpdateStateToConsentOrderMadeAndBulkPrint() {
+    void givenCase_whenNoPension_thenShouldUpdateStateToConsentOrderMadeAndBulkPrint() {
         CallbackRequest callbackRequest =
             doValidCaseDataSetUp(NO_PENSION_VALID_JSON);
         whenServiceGeneratesDocument().thenReturn(caseDocument());
```

Comment:
New function getApprovalDate has been added to this handler.  That can either take caseData with a LocalDate or String and responds with a null or OK string.  Given that, suggest a test.